parsing/ast_mapper.mli

```diff
@@ -177,25 +179,25 @@ val attribute_of_warning: Location.t -> string -> attribute
 
 (** {1 Helper functions to call external mappers} *)
 
-val add_ppx_context_str:
-    tool_name:string -> Parsetree.structure -> Parsetree.structure
+val add_ppx_context_impl:
+    tool_name:string -> Parsetree.implementation -> Parsetree.implementation
 (** Extract information from the current environment and encode it
     into an attribute which is prepended to the list of structure
-    items in order to pass the information to an external
+    items in the implementation in order to pass the information to an external
     processor. *)
 
-val add_ppx_context_sig:
-    tool_name:string -> Parsetree.signature -> Parsetree.signature
-(** Same as [add_ppx_context_str], but for signatures. *)
+val add_ppx_context_intf:
+    tool_name:string -> Parsetree.interface -> Parsetree.interface
+(** Same as [add_ppx_context_impl], but for interfaces. *)
 
-val drop_ppx_context_str:
-    restore:bool -> Parsetree.structure -> Parsetree.structure
-(** Drop the ocaml.ppx.context attribute from a structure.  If
+val drop_ppx_context_impl:
+    restore:bool -> Parsetree.implementation -> Parsetree.implementation
+(** Drop the ocaml.ppx.context attribute from an implementation.  If
     [restore] is true, also restore the associated data in the current
     process. *)
 
-val drop_ppx_context_sig:
-    restore:bool -> Parsetree.signature -> Parsetree.signature
+val drop_ppx_context_intf:
+    restore:bool -> Parsetree.interface -> Parsetree.interface
```

Comment:
Again, I would avoid any change in this file if possible, to make the change less invasive and for compatibility.